### PR TITLE
For EXP-3367: Add epics list to q2 newsletter

### PR DESCRIPTION
**Pull request by eliserichards**
## Description (optional)

- Adds a list of epics that were completed in Q2

## Issue that this pull request resolves (optional)

https://mozilla-hub.atlassian.net/browse/EXP-3367

image

### CODE REVIEW

File: docs/whats-news/2023/2023-q2.mdx [hunk 1]
```diff
@@ -5,7 +5,92 @@ slug: /whats-news/2023-Q2
...
+
+   > Ongoing support for mobile engineers using Nimbus. (jhugman)
+
+## Desktop
+
+💻 [Desktop Localization](https://mozilla-hub.atlassian.net/browse/EXP-3164)
+
+   > Enables localized experiments on the Nimbus Desktop client. (brennie)
+
+💻 [Core Active Behavioral Targeting](https://mozilla-hub.atlassian.net/browse/EXP-2600)
```
Comment (jhugman) [inline docs/whats-news/2023/2023-q2.mdx:46]: Nit: This is for mobile.

File: docs/whats-news/2023/2023-q2.mdx [hunk 2]
```diff
@@ -5,7 +5,92 @@ slug: /whats-news/2023-Q2
...
+
+   > The **Nimbus CLI** (see below) was created to help with first-run testing. First-run is a frequently used, high value surface that requires the experiment configuration to ship with the mobile build, so it is crucial to have support for these experiments to test before shipping. (jhugman)
+
+🍏🤖 [Deprecate Feature Keys in v6 API](https://mozilla-hub.atlassian.net/browse/EXP-3608)
+
+   > To prepare for the switch from single feature to multi-feature, the V6 serializer needed to switch over from using the `feature` key in the branch schema to using the `features` key. Desktop had already switched to `features`, but mobile was still using singular `feature`. This work switches the V6 serializer to always use the `features` key. (brennie)
+
+🍏🤖 [First-run Release Date](https://mozilla-hub.atlassian.net/browse/EXP-3492)
+
+   > Adds a Release Date field for mobile first run experiments to accurately calculate enrollment and durations to save multiple manual calculations. (erichards)
```
Comment (jhugman) [inline docs/whats-news/2023/2023-q2.mdx:34]: :tada:

**Review by jhugman — APPROVED** [review]
This looks great! Thank you! I like the naming of people responsible for the epics!

I have left one comment about Behavioural targeting being in Mobile, rather than desktop.

File: docs/whats-news/2023/2023-q2.mdx (hunk 1 above)
Comment (eliserichards) [reply docs/whats-news/2023/2023-q2.mdx]: oh shoot! thank you!

File: docs/whats-news/2023/2023-q2.mdx [hunk 3]
```diff
@@ -5,7 +5,92 @@ slug: /whats-news/2023-Q2
...
+
+☁️  [Cirrus Documentation](https://mozilla-hub.atlassian.net/browse/EXP-3121)
+
+   > Document/create ADRs for requirements, implementations, and system design diagrams. (ykhurana)
+
+## Experimenter
+
+🙌 [Outreachy contributions](https://mozilla-hub.atlassian.net/browse/EXP-3318)
+
+   > Tasks that were completed by Outreachy intern applicants. (ykhurana)
```
Comment (smazhar789) [inline docs/whats-news/2023/2023-q2.mdx:82]: Could we add the following blurb about the changelog Avishkar worked on: 

With the added capacity from this  internship, we improved the experimenter experience by adding a changelog page for each experimenter.  The primary goal is to introduce a new page that leverages Django, Tailwind CSS, and Alpine.js (new pattern!).

File: docs/whats-news/2023/2023-q2.mdx [hunk 4]
```diff
@@ -5,7 +5,92 @@ slug: /whats-news/2023-Q2
 sidebar_label: What's Newsletter Q2 2023
 ---
 
-## The `nimbus-cli`
+This is a non-exhaustive list of the Nimbus team's accomplishments in Q2, including some deep dives into a few of the more exciting features that have landed.
+
```
Comment (smazhar789) [inline docs/whats-news/2023/2023-q2.mdx:9]: Adding blurb for multifeature:
Enables users to to run experiments that target multiple features at once. We modified experiment creation pages to be able to select multiple features via a multiselect at the top of the page (before editing branches). Each feature value is  validated against its respective schema as before. Modified client and API to support multiple features in each branch and ensure existing experiments that are already published continue to use the single feature branch type without impact.

Comment (smazhar789) [reply docs/whats-news/2023/2023-q2.mdx:9]: epic # https://mozilla-hub.atlassian.net/browse/EXP-1610

File: docs/whats-news/2023/2023-q2.mdx [hunk 5]
```diff
@@ -5,7 +5,100 @@ slug: /whats-news/2023-Q2
...
+## Experimenter
+
+🙌 [Outreachy contributions](https://mozilla-hub.atlassian.net/browse/EXP-3318)
+
+   > Tasks that were completed by Outreachy intern applicants to improve the Experimenter UI. (ykhurana)
+
+🙌  [Outreachy History Tracker Project](https://mozilla-hub.atlassian.net/browse/EXP-3222) [in progress]
+
+   > This work is being completed by our Outreachy intern. This will introduce a new page which show the changelogs of a particualr experiment within the existing project. The primary goal is to introduce a new page that leverages Django, Tailwind CSS, and Alpine.js (new pattern!) to enhance the functionality and user experience of the application. (ykhurana, avi-88)
```
Comment (eliserichards) [inline docs/whats-news/2023/2023-q2.mdx:86]: I added the outreachy project...

File: docs/whats-news/2023/2023-q2.mdx [hunk 6]
```diff
@@ -5,7 +5,100 @@ slug: /whats-news/2023-Q2
...
+
+🔧 [Multifeature](https://mozilla-hub.atlassian.net/browse/EXP-1610)
```
Comment (eliserichards) [inline docs/whats-news/2023/2023-q2.mdx:88]: ... and multi-feature :)

File: docs/whats-news/2023/2023-q2.mdx [hunk 7]
```diff
@@ -5,7 +5,100 @@ slug: /whats-news/2023-Q2
...
+
+   > Enables users to to run experiments that target multiple features at once. We modified experiment creation pages to be able to select multiple features via a multiselect at the top of the page (before editing branches). Each feature value is validated against its respective schema as before. Modified client and API to support multiple features in each branch and ensure existing experiments that are already published continue to use the single feature branch type without impact. (brennie, jlockhart)
```
Comment (eliserichards) [inline docs/whats-news/2023/2023-q2.mdx:90]: ... and multifeature :)

**Pull request merged**